firebase/rules/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "firebase",
```

Comment:
This is should be renamed to `@machinelabs/firebase-rules` as discussed in https://github.com/machinelabs/machinelabs/pull/699#discussion_r164301875

Reply:
Yep, forgot that.